support-frontend/assets/pages/paper-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -112,6 +112,7 @@ function CheckoutForm(props: PropTypes) {
   const days = getDays(props.fulfilmentOption, props.productOption);
   const fulfilmentOptionDescriptor = props.fulfilmentOption === HomeDelivery ? 'Paper' : 'Voucher booklet';
   const fulfilmentOptionName = props.fulfilmentOption === HomeDelivery ? 'Home delivery' : 'Voucher booklet';
+  const deliveryTitle = props.fulfilmentOption === HomeDelivery ? 'Where should we deliver your newspaper' : 'Where should we deliver your vouchers';
```

Comment:
Probably missing a question mark from both of these strings?

Reply:
not a show stopper, but I feel like now we have 3 things which all start with `props.fulfilmentOption === HomeDelivery` we shoud use the rule of 3 and make a PaperOptions struct and put them all inside it, rather than 3 seaprate vals with the same logic